benchmarks/results.md

```diff
@@ -0,0 +1,4 @@
+# Tue Apr 14 2015 09:34:23 GMT-0700 (PDT)
+Todo.create x 110,662 ops/sec ±13.34% (67 runs sampled)
+Todo.find x 76,328 ops/sec ±49.29% (51 runs sampled)
+
```

Comment:
I think we should leave it in VC so we can keep track of the results over time. Like next month when we run it, we can compare it to the this month.